src/main/java/luke/task/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package luke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected static final DateTimeFormatter DATE_TIME_INPUT_FORMATTER
+            = DateTimeFormatter.ofPattern("d/MM/yyyy HH:mm");
+    protected static final DateTimeFormatter DATE_TIME_OUTPUT_FORMATTER
+            = DateTimeFormatter.ofPattern("MMM d yyyy, HH:mm");
```

Comment:
Maybe this can be extracted out to a DateTime class so that the Event class can also use this?